tests/lib/rules/no-unused-prop-types.js

```diff
@@ -2208,6 +2208,54 @@ ruleTester.run('no-unused-prop-types', rule, {
       `,
       settings: {react: {flowVersion: '0.53'}},
       parser: 'babel-eslint'
+    }, {
+      // Issue #1068
+      code: `
+      class MyComponent extends Component {
+        static propTypes = {
+          validate: PropTypes.bool,
+          options: PropTypes.array,
+          value: ({options, value, validate}) => {
+            if (!validate) return;
+            if (options.indexOf(value) < 0)
+              throw new Errow('oops');
+          }
+        }
+
+        render() {
+          return <ul>
+            {this.props.options.map(option =>
+              <li className={this.props.value == option && "active"}>{option}</li>
+            )}
+          </ul>
+        }
+      }
+      `,
+      parser: 'babel-eslint'
+    }, {
+      // Issue #1068
+      code: `
+      class MyComponent extends Component {
+        static propTypes = {
+          validate: PropTypes.bool,
+          options: PropTypes.array,
+          value: function ({options, value, validate}) {
```

Comment:
can you also add a test case for `value({options, value, validate}) {`?

Reply:
👍 Added the test case.